LibNFS.xs

```diff
@@ -802,6 +802,9 @@ static inline void _croak_if_buffer_is_reference( pTHX_ SV* buf_sv ) {
     }
 }
 
+#define _croak_if_uv_exceeds_u32(name, value) \
+    if (value > max) croak("%s: value (%" UVuf ") exceeds maximum (%lu)", name, value, UINT32_MAX);
```

Comment:
Either `max` should be a parameter to the macro or (given the macro name) should be hardcoded to `UINT32_MAX`.

Reply:
Updated.